### PR TITLE
[faq] Added 'advanced osm editing' section

### PR DESCRIPTION
**Pull request by map-per**
Added an 'advanced osm editing' section to the faq, so this page can be linked to in https://github.com/organicmaps/organicmaps/pull/7748.

See also  #203

### CODE REVIEW

File: content/faq/editing/advanced-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-editing/index.md:27]: Let's highlight that ID is for desktops only, and recommend other options for mobile users (they should go before details about how to use ID on the desktop.

File: content/faq/editing/advanced-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-editing/index.md:1]: advanced-map-editing

File: content/faq/editing/advanced-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-editing/index.md:3]: It's better to focus this page on two simple things:
1. Teach new users about OSM.
2. Recommend other editors (focus on mobile ones!). And make a link to another FAQ page "How to fix map errors using web ID editor?", it may have different recommendations, including how to copy a link to the place from OM by clicking on coordinates (there is an osm link there too), and other details.

File: content/faq/editing/advanced-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-editing/index.md:13]: Maybe add a paragraph how/when map data is updated here or below.

File: content/faq/editing/advanced-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-editing/index.md:32]:
```suggestion
2. Browse to the location you want to edit on OpenStreetMap.org and click *Edit*
```

File: content/faq/editing/advanced-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-editing/index.md:31]:
```suggestion
1. Create a new account or log in at [OpenStreetMap.org](https://www.openstreetmap.org)
```

File: content/faq/editing/advanced-editing/index.md
Comment (map-per) [reply content/faq/editing/advanced-editing/index.md:2]: Ok, but I would rather use "How can I do more advanced map editing?".

File: content/faq/editing/advanced-editing/index.md
Comment (map-per) [reply content/faq/editing/advanced-editing/index.md:3]: The problem is, that there are no mobile editors that are good for beginners and can do "advanced" edits (see https://github.com/organicmaps/organicmaps/pull/7748#issuecomment-2035347759). So ID on a desktop PC is the only good option.

File: content/faq/editing/advanced-editing/index.md
Comment (map-per) [reply content/faq/editing/advanced-editing/index.md:32]: > A direct "edit" link to osm can be inserted, right?

Using the OSMCarto map for zooming to the right location first is more convenient. On older devices zooming or moving the map in ID can be very slow on lower zoom levels.

> A hint on how to extract coordinates or osm URL from OM can be helpful

But then you have the coordinates in the clipboard of your phone and not on the desktop PC

File: content/faq/editing/advanced-editing/index.md
Comment (biodranik) [reply content/faq/editing/advanced-editing/index.md:3]: What "advanced" cases for new users, in your opinion, can't be handled with existing mobile editors?

Forcing all mobile users to "find and use desktop to edit the map" doesn't look like a good idea. Most new users won't need really "advanced" stuff, from what we see, users need something simpler, that just can't be added/edited yet in OM.
At least a list of user options is needed for mobiles. I would say, that Vespucci and Go! Map can cover most of the cases.

File: content/faq/editing/advanced-editing/index.md
Comment (biodranik) [reply content/faq/editing/advanced-editing/index.md:32]: We are trying to optimize here the UX for _completely new_ users, not aware of OSM, and who came from the mobile OM app. They need something really _simple_ to get involved. Later, they can learn more about OSM and about any advanced use cases.

File: content/faq/editing/advanced-editing/index.md
Comment (map-per) [reply content/faq/editing/advanced-editing/index.md:3]: I can't say anything about Go! Map, as it's ios only, but Vespucci is rather complicated to use. But mentioning it any way is probably a good idea, as it is the only android editor that can do "advanced" stuff like adding a new road.

Osm Go! and Every Door can only edit POIs and with StreetComplete you can only answer simple quests. So with "advanced" I meant doing more than just editing POIs. So e.g. adding a new road or a building shape.

Have you tried out mobile editors so far? And if yes, how was your experience?

Comment (biodranik) [reply content/faq/editing/advanced-editing/index.md:3]: "Go! Map" is good enough for mobile. Using it to draw ways and fix areas rarely.

Then the text in the app should be clearer: "Simple map editing apps like Every Door or StreetComplete, or more advanced Vespucci" for Android and "Every Door app for simple map edits, and more advanced Go! Map"

File: content/faq/editing/advanced-editing/index.md
Comment (map-per) [reply content/faq/editing/advanced-editing/index.md]: No, I think that rather belongs into a different faq

**Comment by map-per** [comment]
I updated the faq page:
- I added mobile editor, but placed ID first as editing is easier on a bigger screen and with mouse and keyboard. 
- I left everyting in one faq page as scattering the text would make it hard to read

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:18]:
```suggestion
For editing OSM, there are several options. If you have a laptop or desktop computer at hand, it's better to use the [ID Editor](https://www.openstreetmap.org/edit) that runs in your browser. The ID Editor is easy for beginners, and a bigger screen, mouse, and keyboard make map editing easier.
```

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:20]:
```suggestion
For advanced map editing from a mobile device, use [Go!! Map](https://apps.apple.com/us/app/go-map/id592990211) for iOS or [Vespucci](https://play.google.com/store/apps/details?id=de.blau.android) for Android. Go Map is easy for beginners, while Vespucci targets more advanced users.

For simpler edits with more fun, you may also try [Every Door app](https://every-door.app/) for iOS and Android and [StreetComplete app](https://streetcomplete.app/) for Android.
```

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:22]:
```suggestion
```

Following succinct OM style in FAQ is beneficial too )

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:28]:
```suggestion
2. Browse to the location you want to edit on OpenStreetMap.org and click *Edit* at the top
```

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:39]: Can these tutorial hints be embedded above, where these editors are referenced? Or maybe a separate section "Tutorials" would be better?

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:47]: Need to clarify when edits appear in Organic Maps

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:54]: Any live discussion channels? List of telegram/other messenger groups? Remember, our audience is not a geeky one.

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:74]: Is this numbered list properly rendered by Zola?

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [inline content/faq/editing/advanced-map-editing/index.md:68]: It's better to leave an empty line after each header.

**Review by biodranik — COMMENTED** [review]
Thanks! A few comments.

File: content/faq/editing/advanced-map-editing/index.md
Comment (map-per) [reply content/faq/editing/advanced-map-editing/index.md:54]: The help and support page in the OSM Forum is the main place for questions. But there are also other channels (see: https://wiki.openstreetmap.org/wiki/Contact_channels). I haven't tried out any of them though.

File: content/faq/editing/advanced-map-editing/index.md
Comment (map-per) [reply content/faq/editing/advanced-map-editing/index.md]: :+1: I moved the tutorial hints to the editor overview

File: content/faq/editing/advanced-map-editing/index.md
Comment (map-per) [reply content/faq/editing/advanced-map-editing/index.md:66]: Added &nbsp; to have more space between the quote and the next list element

**Comment by map-per** [comment]
Updated the faq page and included your suggestions.

**Comment by biodranik** [comment]
@map-per FYI for the future PRs: with a write access to the repo, if you push a branch here, then its preview can be available for anyone.

**Comment by biodranik** [comment]
We can proceed with the PR to unblock other tasks, although the text already looks too complicated for a non-geeky, non-OSM user. It would be great to stick to the most important, precise, succinct info for new users on the initial page, and move any details for those who are interested to a different page. Otherwise, there is a risk that your grandma just closes the long web page with a lot of text without reading it, and won't proceed with any contributions.

You can test it with OM + a link to this page combo with any (non-geeky) person who didn't know about OM and OSM before.

File: content/faq/editing/advanced-map-editing/index.md
Comment (biodranik) [reply content/faq/editing/advanced-map-editing/index.md:62]: Please don't do that. No visual formatting with the content. It should be done with CSS, I've pushed a fix.

**Comment by map-per** [comment]
I think it's OK that this page explains more difficult content, as it is about **advanced** editing.

For **simple** editing with OM we should create a separate faq page.

**Comment by biodranik** [comment]
There is an important UX flow nuance: _all_ OM users (including complete newbies) will see the related message when they try to add something on the map and won't find the interested object.

That means that the landing page should be easy to digest for _all_ OM users. Those who is interested in more advanced details, may click to read them.

**Comment by map-per** [comment]
> We can proceed with the PR to unblock other tasks

Then let's merge this and do further improvements in the next iterations

**Review by biodranik — APPROVED** [review]
Ok, any ideas on a web page for grannies?

**Comment by map-per** [comment]
We could move community, wiki and data model to a dedicated OpenStreetMap page. And as soon as we have an Organic Maps editing page (for grannies) we could move the OSM Notes section there.

I think, for the FAQ, the Organic Maps editing page is the highest priority now.

**Pull request merged**
